muhammet-tarakci-comma-based.csl

```diff
@@ -0,0 +1,1300 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" page-range-format="chicago">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/#) -->
+  <info>
+    <title xml:lang="tr">Muhammet TARAKCI (Türkçe ve Virgül Ayrımlı Dipnot Sistemi)</title>
+    <title-short xml:lang="tr">muhammet-tarakci-comma-based</title-short>
+    <id>http://www.zotero.org/styles/muhammet-tarakci/comma-based</id>
+    <link href="http://www.zotero.org/styles/muhammet-tarakci" rel="self" xml:lang="tr"/>
```

Comment:
line 8. needs to be the same as the filename and ID., so http://www.zotero.org/styles/muhammet-tarakci-comma-based